src/ECModel.jl

```diff
@@ -657,7 +673,7 @@ function plot_sankey(ECModel::AbstractEC, sank_data::Dict)
         node_colors=sank_data["colors"],
         edge_color=:gradient,
         compact=true,
-        label_size=15,
+        label_size=10,

```

Comment:
We decided that this shall be an argument to the function: label_size=label_size, where label_size was the argument.

The function below at line 702 shall be updated as well to include the label_size option

Reply:
Working on it